### PR TITLE
Validate category title input before delete category

### PR DESCRIPTION
**Pull request by khaledkz**

### CODE REVIEW

File: server/routes/admin/categories/categories.js [hunk 1]
```diff
@@ -41,21 +41,32 @@ router.get('/edit/:categoryId', (req, res) => {
...
 
-router.get('/delete/:categoryId',(req,res)=>{
-    const {categoryId} = req.params;
 
-    const callback=()=>{
-        res.redirect('/admin/categories')
+
+router.post('/delete/:categoryId',(req,res)=>{
+    const {categoryId} = req.params;
+
```
Comment (kabaros) [inline server/routes/admin/categories/categories.js:48]: why did you get rid of `removeCategory` - it seems like a waste to do it in two steps?

File: server/public/stylesheets/style.css [hunk 2]
```diff
@@ -37,8 +37,39 @@ nav#serverNav{
   border-left: none;
   vertical-align: middle;
   
-  
  }
+ 
+ .validation{
+  width:90%;
+    height: 300px;
```
Comment (kabaros) [inline server/public/stylesheets/style.css]: fix indentation

File: server/routes/admin/categories/categories.js (hunk 1 above)
Comment (khaledkz) [reply server/routes/admin/categories/categories.js:48]: it's check the name of title before delete the category so we don't need to delete the category directly

File: server/public/stylesheets/style.css (hunk 2 above)
Comment (khaledkz) [reply server/public/stylesheets/style.css]: fixed

File: server/routes/admin/categories/categories.js (hunk 1 above)
Comment (khaledkz) [reply server/routes/admin/categories/categories.js:48]: image

**Comment by kabaros** [comment]
@kharazzoro could you fix the conflict too?

**Comment by khaledkz** [comment]
Updated

**Review by kabaros — APPROVED** [review]

**Pull request merged**